Sources/UIComponent/Components/FlexLayout.swift

```diff
@@ -99,7 +99,7 @@ extension FlexLayoutComponent {
           let child = children[index]
           if let child = child as? Flexible {
             let crossReserved = crossPerFlex * child.flex + cross(renderers[index].size)
-            let constraint = Constraint(minSize: size(main: alignItems == .stretch ? main(lineSize) : 0, cross: crossReserved),
+            let constraint = Constraint(minSize: size(main: (alignItems == .stretch || child.alignSelf == .stretch) ? main(lineSize) : 0, cross: crossReserved),
```

Comment:
If the child has `alignSelf = start` while `alignItems = .stretch`, then you have an error here.
Let's make it a local variable:
```swift
let alignChild = child.alignSelf ?? alignItems
```

Reply:
👌🏻, now corrected